internal/venice-common/src/main/java/com/linkedin/venice/pubsub/adapter/SimpleProduceResultImpl.java

```diff
@@ -0,0 +1,53 @@
+package com.linkedin.venice.pubsub.adapter;
+
+import com.linkedin.venice.pubsub.api.ProduceResult;
+
+
+/**
+ * A simple implementation of ProduceResult interface for testing purposes.
+ */
+public class SimpleProduceResultImpl implements ProduceResult {
+  private final String topic;
+  private final int partition;
+  private final long offset;
+  private final int serializedKeySize;
+  private final int serializedValueSize;
+
+  public SimpleProduceResultImpl(
```

Comment:
this is fine, but I guess ApacheKafkaProducerResult can also achieve the same testing purposes though...it is fine to keep it but maybe using ApacheKafkaProducerResult directly gives you more test coverage :)

Reply:
That will potentially start leaking Kafka APIs such as `RecordMetadata` in the rest of the code base again.